apps/src/componentLibrary/textField/TextField.tsx

```diff
@@ -21,6 +21,8 @@ export interface TextFieldProps extends AriaAttributes {
   name: string;
   /** The value attribute specifies the value of an input element. */
   value?: string;
+  /** Can be specified password if needed. */
+  type?: string;
```

Comment:
I think it's better to rename this prop into `inputType` since `type` might lead to some confusions/misunderstading in the future, giving that in other DSCO components we're using `type` prop mostly for styles related logic

Reply:
Also, I do think we need to specify all possible `inputType` values here, cause there're a lot of input types in html and not all of them will work here. `'text' | 'email' | 'password' | 'number'` should work I think, please let me know if there're any other `inputTypes` we might need for this component that I've missed.

Reply:
I removed the updates from this PR and will tag you in the new one!